lib/spack/spack/spec.py

```diff
@@ -147,14 +147,14 @@
 
 #: Default format for Spec.format(). This format can be round-tripped, so that:
 #:     Spec(Spec("string").format()) == Spec("string)"
-default_format = (
+DEFAULT_FORMAT = (
```

Comment:
Is there a reason for this change? It introduces a bunch of orthogonal changes into the diff for this PR

Reply:
I'll extract that. The reason is to make apparent where we are using global constants in the code.

Reply:
See #39573